### ISSUE TITLE
MacOSX (in)compatibility

### ISSUE THREAD

**Issue by HugoNebreda** [vanila-io/wireflow#33]
Hi!

I stumbled upon your project and I think it has great potential

Tried the online version with lots of hope but no luck, and then tried to execute it locally on my MacBook (adapting Linux instructions) but no luck either

I don't know if there is any incompatibility by default with MacOSX, or the project is simply discontinued

Thanks

**Comment by entrptaher** [comment]
Hello, have you tried the docker from this comment https://github.com/vanila-io/wireflow/issues/26#issuecomment-455940286?

**Comment by shtefcs** [comment]
hey @HugoNebreda online version should work, but the servers might go down time to time. MAybe u got it in that wrong moment. However, Wireflow was just a prototype and its bit buggy. But we are rebuilding it from scratch. 

Stay tuned for upcoming period.

**Comment by HugoNebreda** [comment]
Hi @entrptaher

I tried the docker solution, but can't finish building the container.

There is an error when running:
`RUN meteor yarn --ignore-optional`

And the error is:
`error fibers@4.0.2: The engine "node" is incompatible with this module. Expected version ">=10.0.0". Got "8.8.1"`

Tried to update node version on my host, and also changed the `FROM node:8` to `node:10` but cannot pass from there

Any idea?
